### PR TITLE
Detection list performance

### PR DESCRIPTION
**Pull request by jkburges**
I intend to test this against a production copy, however I'm putting it up so that the "delete materialised view" task can be started (hopefully this afternoon while I'm still here to help out).

Perhaps hold of on merging until I do the prod copy testing, but reviews welcome now.

### CODE REVIEW

File: grails-app/conf/Config.groovy [hunk 1]
```diff
@@ -318,3 +322,71 @@ rawDetection.extract.view.select = '''select timestamp, to_char((timestamp::time
...
+            JOIN device receiver ON receiver_deployment.receiver_id = receiver.id
+            JOIN device_model ON receiver.model_id = device_model.id
+            JOIN organisation ON receiver.organisation_id = organisation.id
+            JOIN receiver_download_file ON valid_detection.receiver_download_id = receiver_download_file.id'''
+
+        detection_view {
+            definition = """SELECT ${columns}
+            FROM valid_detection
+            ${receiver_joins}
+            LEFT OUTER JOIN sec_user ON receiver_download_file.requesting_user_id = sec_user.id
```
Comment (jonescc) [inline grails-app/conf/Config.groovy]: LEFT OUTER JOIN is the same as LEFT JOIN (OUTER keyword is optional)

**Comment by jkburges** [comment]
This seems to perform as expected against a prod DB copy on yolo.

**Comment by jonescc** [comment]
Looks good to me.

**Comment by julian1** [comment]
Merging this on the basis of jkburges and jonescc tests. 

Please don't deploy to 3-nsp.

**Pull request merged**